src/grpcHandler.cpp

```diff
@@ -0,0 +1,221 @@
+    std::cout << "Failed to read SSL certificate" << filename << std::endl;
+  }
+	return;
+}
+
+void grpcHandler::RunServer(std::shared_ptr<VssCommandProcessor> Processor, std::string certPath) {
+  string server_address("127.0.0.1:50051");
+  RequestServiceImpl service;
+
+  grpc::EnableDefaultHealthCheckService(true);
+  grpc::reflection::InitProtoReflectionServerBuilderPlugin();
+  ServerBuilder builder;
+
+  std::string key;
+	std::string cert;
+  std::string root;
+  std::string serverkeyfile_= certPath + "/Server.key";
+  std::string serverpemfile_= certPath + "/Server.pem";
+  std::string serverrootfile= certPath + "/CA.pem";
+  read(serverkeyfile_, cert);
+	read(serverpemfile_, key);
+  read(serverrootfile, root);
+
+  grpc::SslServerCredentialsOptions::PemKeyCertPair keycert{
+    cert,
+    key
+  };
+
+	grpc::SslServerCredentialsOptions sslOps;
+  sslOps.pem_root_certs = root;
+	sslOps.pem_key_cert_pairs.push_back(keycert);
+
+	builder.AddListeningPort(server_address, grpc::SslServerCredentials(sslOps));
+
+  // Register "service" as the instance through which we'll communicate with
+  // clients. In this case it corresponds to an *synchronous* service.
+  builder.RegisterService(&service);
+  // Finally assemble the server.
+  std::shared_ptr<Server> server(builder.BuildAndStart());
+  handler.grpcServer = server;
+  handler.grpcProcessor = Processor;
+  cout << "Kuksa viss grpc server Version 1.0.0" << endl;
+  cout << "Server listening on " << server_address << endl;
+
+  // Wait for the server to shutdown. Note that some other thread must be
+  // responsible for shutting down the server for this call to ever return.
+  handler.grpcServer->Wait();
+}
+
+class GrpcConnection {
+ public:
+  GrpcConnection(std::shared_ptr<Channel> channel)
+      : stub_(viss_client::NewStub(channel)) {}
+  // get the command from the terminal and request the server
+  std::string HandleRequest(){
+    // Container for the data we send to the server.
+    CommandRequest request;
+    // Container for the data we expect from the server.
+    CommandReply reply;
+    std::string command;
+    // Get command input from input
+    std::getline(std::cin,command);
+    request.set_command(command);
+    ClientContext context;
+    // The actual RPC
+    Status status = stub_->HandleRequest(&context, request, &reply);
+    if (status.ok()) {
+      return reply.message();
+    } else {
+      std::cout << status.error_code() << ": " << status.error_message()
+                << std::endl;
+      return "RPC failed";
+    }
+  }
+ private:
+  std::unique_ptr<viss_client::Stub> stub_;
+};
+
+void grpcHandler::startClient(std::string port, std::string certPath){
+  std::string cert; 
+	std::string key;
+  std::string root;
+  std::string clientpemfile_ = certPath + "/Client.pem";
+  std::string clientkeyfile_ = certPath + "/Client.key";
+  std::string clientrootfile_ = certPath + "/CA.pem";
+  read (clientpemfile_, cert);
+	read (clientkeyfile_, key);
```

Comment:
```suggestion
  read (clientkeyfile_, key);
```

pls replace `\t` to `  `